src/main/java/org/jitsi/videobridge/octo/OctoRelayService.java

```diff
@@ -68,17 +85,30 @@ public void start(BundleContext bundleContext)
 
         try
         {
-            relay = new OctoRelay(address, port);
-            relay.setPublicAddress(publicAddress);
-            bundleContext
-                .registerService(OctoRelayService.class.getName(), this,
-                                 null);
+            udpTransport = new UdpTransport(address, port, logger, OCTO_SO_RCVBUF, OCTO_SO_SNDBUF);
+            logger.info("Created Octo UDP transport");
         }
         catch (UnknownHostException | SocketException e)
         {
-            logger.error("Failed to initialize Octo relay with address "
-                             + address + ":" + port + ". ", e);
+            logger.error("Failed to initialize Octo UDP transport with " +
+                "address " + address + ":" + port + ".", e);
+            return;
         }
+
+        octoTransport = new OctoTransport(publicAddress + ":" + port, logger);
+        logger.info("Created OctoTransport");
```

Comment:
Move that log to `OctoTransport` so  it can print it's ID?

Reply:
Good idea, fixed.